test/integration/cli/config.test.js

```diff
@@ -50,7 +50,7 @@ describe('CLI config', () => {
 		// the page title, so reading the title confirms that headers were sent by Pa11y
 		it('tests the page using the specified HTTP method', () => {
 			assert.isArray(pa11yResponse.json);
-			assert.lengthEquals(pa11yResponse.json, 1);
+			assert.lengthEquals(pa11yResponse.json, 13);
```

Comment:
Theres a lot of tests cases in here where the numbers have increased significantly. Is that a result of `HMTLCS.js` picking up more problems now? Or are we now not passing options to it correctly so we're getting back different results.

Reply:
More issues being found by html_codesniffer